### ISSUE TITLE
Old deprecated icons

### ISSUE THREAD

**Issue by NilTPL** [catppuccin/rofi#19]
Hi - after multiple days of troubleshooting what was wrong with my fonts and why the icons did not load, ive found out that the icons themselves specified in the config.rasi file are old icons that have been since removed from the fonts themselves.

**Comment by ForeverZer0** [comment]
I had the same issue, and I was foolishly searching through "config" folder, taking an embarrassing amount of time before realizing it was configured in the `bin/powermenu` script. Though it was worthwhile finding/changing some other hard-coded font settings that didn't fit my aesthetic, so it wasn't completely wasted time.

It might be worth considering moving all of the hard-coded fonts/icons to somewhere more obvious, such as the `fonts.rasi`. From what I can tell, the font specified there is overridden in multiple locations in files that reference it.

**Comment by nonetrix** [comment]
See this issue
https://github.com/catppuccin/rofi/pull/17

**Comment by IQubic** [comment]
I would just like to say that this issue is present on my NixOS machine. I'm viewing `basic/.config/rofi/config.rasi` on GitHub in Firefox, and it looks as shown below. I'm currently using `Hack NerdFont` here (I've set it in Firefox's settings) and it's not working. I tried copying the contents of that file to a temp file and editing it in a few different text editors and they all look similar.
![2024-04-14_18-34-32](https://github.com/catppuccin/rofi/assets/22628816/ce16d263-305e-4d01-9fb3-3cfe5e05dbb9)
